PrebidMobile/AdUnits/AdUnit.swift

```diff
@@ -147,6 +147,12 @@ public class AdUnit: NSObject, DispatcherDelegate {
             return
         }
         
+        if let auctionSettingsId = Prebid.shared.auctionSettingsId,
+           auctionSettingsId.isEmpty || auctionSettingsId.containsOnly(.whitespaces) {
+            completion(BidInfo(resultCode: .prebidInvalidAuctionSettingsId))
+            return
```

Comment:
Is it necessary for the SDK to fail the entire `fetchDemand` process when the optional parameter is invalid? I would suggest logging the error in this case and proceeding with the bid request with the `prebidServerAccountId` if present.